### PR TITLE
Create IsoMSX+ tileset folder

### PR DESCRIPTION
**Pull request by Zupercharged**
#### Summary
Infrastructure "Create repo for iso MSX conversion"

#### Content of the change
Creation of new IsoMSX+ repository folder with blank readme.txt, for future population with completed isometrically converted MSX+ assets

### CODE REVIEW

**Comment by Fris0uman** [comment]
Uploading some sprites with it would probably be better

**Comment by Zupercharged** [comment]
submitted an initial set of folders with sprite id's and .json referenced from existing tilesets

File: gfx/IsoMSX+/pngs_furniture_32x32/f_autodoc/f_autodoc.json [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "id": "f_autodoc",
+  "fg": [
+    "f_autodoc_ne",
+    "f_autodoc_se",
+    "f_autodoc_sw",
+    "f_autodoc_nw"
+  ],
+  "rotates": true,
```
Comment (Fris0uman) [inline gfx/IsoMSX+/pngs_furniture_32x32/f_autodoc/f_autodoc.json:9]: remove the comma here

File: gfx/IsoMSX+/pngs_furniture_32x32/f_centrifuge/f_centrifuge.json [hunk 2]
```diff
@@ -0,0 +1,12 @@
+[
+{
+  "id": "f_centrifuge",
+  "fg": "f_centrifuge",
```
Comment (Fris0uman) [inline gfx/IsoMSX+/pngs_furniture_32x32/f_centrifuge/f_centrifuge.json:4]: no need for json if the "id" and "fd" field are the same

File: gfx/IsoMSX+/pngs_furniture_tall_32x64/f_robotic_arm.json [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "id": "f_robotic_arm",
```
Comment (Fris0uman) [inline gfx/IsoMSX+/pngs_furniture_tall_32x64/f_robotic_arm.json:2]: same id and fd, see previous comment

File: gfx/IsoMSX+/pngs_large_64x64/f_autodoc_couch.json [hunk 4]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "f_autodoc_couch",
+  "fg": "f_autodoc_couch",
```
Comment (Fris0uman) [inline gfx/IsoMSX+/pngs_large_64x64/f_autodoc_couch.json:3]: can remove

File: gfx/IsoMSX+/pngs_large_64x64/tr_portal.json [hunk 5]
```diff
@@ -0,0 +1,5 @@
+{
+  "id": "tr_portal",
```
Comment (Fris0uman) [inline gfx/IsoMSX+/pngs_large_64x64/tr_portal.json:2]: can remove

File: gfx/IsoMSX+/tile_info.json [hunk 6]
```diff
@@ -0,0 +1,23 @@
+[
+  {
+    "pixelscale": 1,
+      "height": 16,
+      "iso": true,
+      "width": 32
+  },
+  {
+    "furniture.png": { "sprite_width": 32, "sprite_height": 32 }
```
Comment (Fris0uman) [inline gfx/IsoMSX+/tile_info.json:9]: Probably should name tilesheet according to their size, we might make 32x32 sprite that are not furnitures at some point for exemple

Comment (Zupercharged) [reply gfx/IsoMSX+/tile_info.json:9]: I based this on the MShockXotto+ repository, where the sprite folders are named pngs_tiles_32x32 or pngs_tall _32x64 but the tilesheets they compile into are simply noted as tiles.png or tall.png in the tile_info.json, omitting the specified sprite size.

Is this naming convention not necessary for the tileset compiler to work?

**Pull request merged**